docs/src/main/asciidoc/_admin-guide/subsystem-configuration/MicroProfile_Metrics.adoc

```diff
@@ -35,9 +35,9 @@ standalone configurations explicitly sets it to `false` to accept unauthenticate
 The Metric HTTP endpoint is accessible on WildFly HTTP management interface http://localhost:9990/metrics[http://localhost:9990/metrics].
 
 Secured access to the HTTP endpoint is controlled by the `security-enabled` attribute of the `/subsystem=microprofile-metrics-smallrye` resource.
-If it is set to `true`, the HTTP client must be authenticated.
+The value of this attribute will override the `security-enabled` attribute of the `/subsystem=metrics` resource (documented in <<metrics-http-endpoint,Metrics subsystem configuration guide>>).
```

Comment:
What about `exposed-subsystems` and `prefix` does same overriding logic apply as well in case of these properties.

If `metrics` subsystem is responsible for registring /metrics http endpoint and will be always present, do we really need this overriding logic?

Reply:
`exposed-subsystems` and `prefix` are specific to the `microprofile-metrics-smallrye` and do not provide this overriding logic.

This special case to handle the `security-enabled` is to keep it backwards compatible. Users that change this value expects to be able to use it. Making it a no-op would have break their configuration.